lib/scraper.js

```diff
@@ -99,6 +99,10 @@ Scraper.prototype.getResourceHandler = function getHandler (resource) {
 Scraper.prototype.loadResource = function loadResource (resource) {
 	var self = this;
 
+	if(!self.options.urlFilter(resource.url)){
+		return Promise.resolve(null);
```

Comment:
I'm not sure but I think we can return rejected promise so modifications in file-handlers will be not needed